ddp/ddp_pipeline/src/main/java/org/mskcc/cmo/ks/ddp/pipeline/util/DDPUtils.java

```diff
@@ -166,6 +166,21 @@ public static String resolvePatientCurrentAge(DDPCompositeRecord compositeRecord
         return anonymizePatientAge(age.intValue());
     }
 
```

Comment:
Maybe add a javadoc here to save our future selves (or future victims) a headache :)